src/styles/main.scss

```diff
@@ -1,3 +1,23 @@
-body {
+@import "./utils/variables";
+@import "./utils/functions";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/main.scss";
+@import "./blocks/catalog.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body,
+ul,
+h2,
+p {
   margin: 0;
 }
```

Comment:
for resets better to make another new file, and there write all resets, so that when we will have over 50 files, it will be easier to find our resets